### ISSUE TITLE
Filter based on DNS rcode

### ISSUE THREAD

**Issue by sevencastles** [dmachard/dns-collector#32]
Hi @dmachard,

Could you please extend the filtering feature to exclude/include replies based on the DNS return code? For example, I would like to filter out all the NOERRORs.

Thank you very much in advance,
Carlo

**Comment by dmachard** [comment]
a new beta release is available [0.15.0-b1](https://github.com/dmachard/dns-collector/releases/tag/v0.15.0-b1). Any feedback will be appreciated

The feature (rcode exclude only) can be activated with the following example:

```
subprocessors:
  filtering:
    # drop response according to the return code (NOERROR, ...). This list is empty by default
    drop-rcodes:
      - NOERROR
```

**Comment by dmachard** [comment]
new release v0.15.0
